TEST_APPS/device/nfcapp/SmartPoster.cpp

```diff
@@ -0,0 +1,284 @@
+/* mbed Microcontroller Library
```

Comment:
smart poster class is only for testing? shouldn't it be available for normal usage?

Reply:
That would be useful to have a Smart Poster class in the codebase however we should do design review of the example one and see if it can fit the mbed os codebase.